core/src/test/scala/org/broadinstitute/dsde/rawls/deltalayer/MockDeltaLayerWriter.scala

```diff
@@ -1,5 +1,10 @@
 package org.broadinstitute.dsde.rawls.deltalayer
+import akka.http.scaladsl.model.Uri
+import org.broadinstitute.dsde.rawls.model.DeltaInsert
 
-class MockDeltaLayerWriter extends DeltaLayerWriter {
+import scala.concurrent.Future
 
+class MockDeltaLayerWriter extends DeltaLayerWriter {
+  // TODO: flesh out for unit tests, ideally using a mocked Google storage service
+  override def writeFile(writeObject: DeltaInsert): Future[Uri] = ???
```

Comment:
there exist spots in the test code that need a `DeltaLayerWriter` in a constructor, but don't ever use it - so here's a mock with zero implementation